Traceback (most recent call last):
  File "/Users/henrique/Me/dojogameplay/happynumbers/happynumbers.py", line 4, in <module>
    assert happy(1) == True
TypeError: 'NoneType' object is not callable
